AUTHORS

```diff
@@ -0,0 +1,2 @@
+Erez Freiberger <erez@epsagon.com>
```

Comment:
Use dev@epsagon.com and support@epsagon.com in here

Reply:
it is autogenerated by PBR by the commits

Reply:
👍